cmd/godmine/main.go

```diff
@@ -238,11 +240,14 @@ func getConfig() config {
 
 func addIssue(subject, description string) {
 	var issue redmine.Issue
-	c := redmine.NewClient(conf.Endpoint, conf.Apikey)
+	c, err := redmine.NewClient(conf.Endpoint, auth)
```

Comment:
🚫 **[checkstyle]** <staticcheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>SA1019: redmine.NewClient is deprecated: Use redmine.ClientBuilder to create a redmine client that supports more options and detects configuration mistakes.